fe1-web/components/QRCode.tsx

```diff
@@ -14,33 +13,30 @@ const QRCode = (props: IPropTypes) => {
   const identity = KeyPairStore.getPublicKey().toString();
   const { visibility } = props;
   const { size } = props;
-  const { text } = props;
 
   // Displays a QR code with the Public key of the current user
   return (visibility)
     ? (
-      <>
-        <TextBlock text={String(text)} />
-        <View style={[styleContainer.topCenter, { padding: 10 }]}>
-          <QRCodeDisplay value={identity} size={Number(size)} />
-        </View>
-      </>
+      <View style={[styleContainer.anchoredCenter, { padding: 10, justifyContent: 'flex-start' }]}>
+        <QRCodeDisplay value={identity} size={Number(size)} />
+      </View>
```

Comment:
You don't need the explicit cast anymore since you defined `IPropTypes` manually :)